cassandra/client.go

```diff
@@ -322,21 +331,23 @@ func addSslOptions(cluster *gocql.ClusterConfig, options *sslOptions) *gocql.Clu
 	return cluster
 }
 
-func initializeSession(cluster *gocql.ClusterConfig) *gocql.Session {
+func initializeSession(cluster *gocql.ClusterConfig, co clientOptions) *gocql.Session {
 	session, err := cluster.CreateSession()
 	if err != nil {
 		log.Fatal(err.Error())
 	}
 
-	if err := session.Query(createKeyspaceCQL).Exec(); err != nil {
-		log.Fatal(err.Error())
+	if co.createKeyspace {
```

Comment:
@candysmurf you cannot expect that your client will have this permissions. And as far as we cannot do that, we need this flag to run tasks for enterprise ready cluster.